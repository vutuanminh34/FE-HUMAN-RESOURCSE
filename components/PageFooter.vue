<template>
  <v-footer height="auto" class="white pa-3 app--footer">
    <span class="caption">Hybrid Technologies &copy; 2020</span>
    <v-spacer />
    <v-icon color="pink" small> mdi-heart </v-icon>
  </v-footer>
</template>
