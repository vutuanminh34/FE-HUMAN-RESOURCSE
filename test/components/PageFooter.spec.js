import Vue from 'vue'
import Vuetify from 'vuetify'

// Components
import PageFooter from '@/components/PageFooter'

// Utilities
import { mount, createLocalVue } from '@vue/test-utils'

Vue.use(Vuetify)
const localVue = createLocalVue()

describe('PageFooter.vue', () => {
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })

  it('should have a custom title and match snapshot', () => {
    const wrapper = mount(PageFooter, {
      localVue,
      vuetify,
      propsData: {},
    })

    // With jest we can create snapshot files of the HTML output
    expect(wrapper.html()).toMatchSnapshot()

    // We could also verify this differently
    // by checking the text content
    const title = wrapper.find('span.caption')

    expect(title.text()).toBe('Hybrid Technologies Â© 2020')
  })
})
