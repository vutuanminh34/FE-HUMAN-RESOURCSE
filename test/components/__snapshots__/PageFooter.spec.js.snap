// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageFooter.vue should have a custom title and match snapshot 1`] = `
"<footer class=\\"v-footer white pa-3 app--footer v-sheet theme--light\\"><span class=\\"caption\\">Hybrid Technologies Â© 2020</span>
  <div class=\\"spacer\\"></div> <i aria-hidden=\\"true\\" class=\\"v-icon notranslate mdi mdi-heart theme--light pink--text\\" style=\\"font-size: 16px;\\"></i></footer>"
`;
